.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: required
+
+language: node_js
+node_js:
+  - "8"
+  - "9"
+services:
+  - docker
+before_install:
+  - npm i -g npm@latest
+  - npm i -g yarn
+install:
+  - yarn
+script:
```

Comment:
probably you'll need to lunch docker-compose before running the tests? 
Or at least install rabbitmq in the vm

Reply:
There is a docker-compose file which just needs `docker-compose up -d` to be run before running tests. I believe travis accepts this config for docker containers

```yml
services:
  - docker
```